### ISSUE TITLE
Notification support for Alertzy

### ISSUE THREAD

**Issue by idaanx** [louislam/uptime-kuma#4398]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification

### 🔖 Feature description

Add support for https://alertzy.app, which works like Pushover but is free for 100 messages a day. The API is also very similar to Pushover so shouldn't be hard to implement.

### ✔️ Solution

Basically a copy of Pushover but with sightly tweaked paramaters according to the documentation https://alertzy.app/#integration

### 📝 Additional Context

I've tried using the webhook but this does not have the necessary control.

**Comment by CommanderStorm** [comment]
This is kind of niche/uncommon notification provider and 100 free push messages seems quite limted.
The project also seems quite abandoned looking at https://github.com/ryanjones/BrassSheepdog or https://twitter.com/alertzy

If you/someone else would like to contribute such a provider, here is the Contribution guide:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md 

If you do decide to do this, please not these two suggestions, which often push PRs back by a review-cycle:
- Ensure that the PR includes screenshots of the `DOWN`/`UP` and `testing`/`cert-expiry` events.
- All keys that can/should be translated are in `en.json` (⇒ our translators can translate this and no merge-confilicts between weblate and master can arise)

**Comment by idaanx** [comment]
The links you provided is a different sms service which is abandoned, but uses the same name. This Alertzy.app has been updated between 2020 and 2023 according to the iOS update history.

Also a 100 messages per day is more than enough for me, not sure how much others need...

If someone could implement this service it would be highly appreciated.
